Source/WebCore/dom/Document.h

```diff
@@ -321,7 +321,7 @@ enum class NodeListInvalidationType : uint8_t {
     InvalidateOnHRefAttrChange,
     InvalidateOnAnyAttrChange,
 };
-const uint8_t numNodeListInvalidationTypes = static_cast<uint8_t>(NodeListInvalidationType::InvalidateOnAnyAttrChange) + 1;
+const uint8_t numNodeListInvalidationTypes = enumToUnderlyingType(NodeListInvalidationType::InvalidateOnAnyAttrChange) + 1;
```

Comment:
would be nice to use `std::underlying_type_t<NodeListInvalidationType>` instead of `uint8_t` for the variable then, or `auto`

Reply:
Fixed. Uses `auto'.